### PR TITLE
.NET Framework -> .NET (/standard/collections)

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #21191 .

This PR is for standard/asynchronous* through standard/collections/*.

### CODE REVIEW

File: docs/standard/asynchronous-programming-patterns/interop-with-other-asynchronous-patterns-and-types.md [hunk 1]
```diff
@@ -6,30 +6,24 @@ dev_langs:
...
-- [Tasks and wait handles](#WaitHandles) ([from wait handles to TAP](#WHToTap) or [from TAP to wait handles](#TapToWH))  
-  
-<a name="APM"></a>
-## Tasks and the Asynchronous Programming Model (APM)  
+
+A brief history of asynchronous patterns in .NET:
+
+- .NET Framework 1.0 introduced the <xref:System.IAsyncResult> pattern, otherwise known as the [Asynchronous Programming Model (APM)](asynchronous-programming-model-apm.md), or the `Begin/End` pattern.
+- .NET Framework 2.0 added the [Event-based Asynchronous Pattern (EAP)](event-based-asynchronous-pattern-eap.md).
+- .NET Framework 4 introduced the [Task-based Asynchronous Pattern (TAP)](task-based-asynchronous-pattern-tap.md), which supersedes both APM and EAP and provides the ability to easily build migration routines from the earlier patterns.
```
Comment (tdykstra) [inline docs/standard/asynchronous-programming-patterns/interop-with-other-asynchronous-patterns-and-types.md:21]: Big improvement :smile:

File: docs/standard/base-types/standard-timespan-format-strings.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ The following table lists the standard time interval format specifiers.
...
 |*fffffff*|The optional fractional portion of a second.  Its value can range from "0000001" (one tick, or one ten-millionth of a second)  to "9999999" (9,999,999 ten-millionths of a second, or one second less one tick).|  
   
- Unlike the "g" and "G" format specifiers, the "c" format specifier is not culture-sensitive. It produces the string representation of a <xref:System.TimeSpan> value that is invariant and that is common to all previous versions of the .NET Framework before the .NET Framework 4. "c" is the default <xref:System.TimeSpan> format string; the <xref:System.TimeSpan.ToString?displayProperty=nameWithType> method formats a time interval value by using the "c" format string.  
+ Unlike the "g" and "G" format specifiers, the "c" format specifier is not culture-sensitive. It produces the string representation of a <xref:System.TimeSpan> value that is invariant and that's common to .NET Framework 4 and earlier versions. "c" is the default <xref:System.TimeSpan> format string; the <xref:System.TimeSpan.ToString?displayProperty=nameWithType> method formats a time interval value by using the "c" format string.
```
Comment (tdykstra) [inline docs/standard/base-types/standard-timespan-format-strings.md:60]: the original here sounds like it says "all versions earlier than 4" but the rewrite says "4 and earlier versions"

**Review by tdykstra — APPROVED** [review]
LGTM, one thing to take a second look at.

File: docs/standard/base-types/standard-timespan-format-strings.md (hunk 2 above)
Comment (gewarren) [inline docs/standard/base-types/standard-timespan-format-strings.md:60]:
```suggestion
 Unlike the "g" and "G" format specifiers, the "c" format specifier is not culture-sensitive. It produces the string representation of a <xref:System.TimeSpan> value that is invariant and that's common to versions prior to .NET Framework 4. "c" is the default <xref:System.TimeSpan> format string; the <xref:System.TimeSpan.ToString?displayProperty=nameWithType> method formats a time interval value by using the "c" format string.  
```

**Pull request merged**